Microsoft.Toolkit.Uwp.UI.Controls/CameraPreview/CameraPreview.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style x:Name="MessageStyle" TargetType="TextBlock">
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontWeight" Value="Bold"/>
+        <Setter Property="Foreground" Value="Red"/>
+    </Style>
+    <Style TargetType="local:CameraPreview" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:CameraPreview">
+                    <Grid Background="{TemplateBinding Background}">
+                        <MediaPlayerElement x:Name="MediaPlayerElementControl" MinWidth="300" Width="400" HorizontalAlignment="Left">
+                        </MediaPlayerElement>
+                        <Button x:Name="FrameSourceGroupButton" Background="Transparent" 
+                                VerticalAlignment="Top" HorizontalAlignment="Left" Margin="5">
+                            <Image Source="{TemplateBinding FrameSourceGroupButtonIcon}" Width="25"></Image>
```

Comment:
Use a FontIcon instead of Image. Recommend this Segoe MDL2 icon:

![image](https://user-images.githubusercontent.com/711864/39460227-a54b9776-4cb6-11e8-88f3-e95f29ed8ddc.png)

Reply:
Changed to use font icon instead.